README.md

```diff
@@ -27,6 +27,74 @@ Required depot (and manifest, optionally):
 7. Double click `start_tf2.bat`.
 8. Enjoy!
 
+## Build
+
+DISCLAIMER: This is the big kids zone. If you are not a professional, building the game from source is not what you want. Use the pre-built [Releases](https://github.com/mastercomfig/team-comtress-2/releases). Also, building this on Mac/Linux, while possible, is not covered here. It might be much more complicated (or not)
+
+### Setup
+1. Get [.Net Core Runtime](https://dotnet.microsoft.com/download) (for Depot Downloader, latest version is fine)
+1. Get [Visual Studio 2019 Community Edition](https://visualstudio.microsoft.com/vs/) (for building TF2)

```

Comment:
Might be good to mention that the workload is Desktop development with C++, along with C++ MFC optional component.